static/scss/email-composition-dialog.scss

```diff
@@ -40,9 +40,78 @@
           height: auto;
           border: 1px solid $border-color;
           border-radius: 2px;
+          max-height: 50vh;
 
           .public-DraftEditor-content {
             padding-left: 10px;
+            color: black;
+            font-weight: normal;
+            font-family: arial;
+
+            // this is an attempt at a hacky default-ish display for hX tags
+            h1 {
+              display: block;
+              font-size: 2em;
+              margin-top: 0.67em;
+              margin-bottom: 0.67em;
+              margin-left: 0;
+              margin-right: 0;
+              font-weight: bold;
+            }
+
+            h2 {
+              display: block;
+              font-size: 1.5em;
+              margin-top: 0.83em;
+              margin-bottom: 0.83em;
+              margin-left: 0;
+              margin-right: 0;
+              font-weight: bold;
+            }
+
+            h3 {
+              display: block;
+              font-size: 1.17em;
+              margin-top: 1em;
+              margin-bottom: 1em;
+              margin-left: 0;
+              margin-right: 0;
+              font-weight: bold;
+            }
+
+            h4 {
+              display: block;
+              margin-top: 1.33em;
+              margin-bottom: 1.33em;
+              margin-left: 0;
+              margin-right: 0;
+              font-weight: bold;
+              font-size: 1em;
+            }
+
+            h5 {
+              display: block;
+              font-size: .83em;
+              margin-top: 1.67em;
+              margin-bottom: 1.67em;
+              margin-left: 0;
+              margin-right: 0;
+              font-weight: bold;
+            }
+
+            h6 {
+              display: block;
+              font-size: .67em;
+              margin-top: 2.33em;
+              margin-bottom: 2.33em;
+              margin-left: 0;
+              margin-right: 0;
+              font-weight: bold;
+            }
+
+            h1, h2, h3, h4, h5, h6 {
+              padding: 0;
```

Comment:
Could you refactor `margin-left`, `margin-right`, `display`, and `font-weight` from the individual style defs above into this one since they're all the same?